tests/fparser/autodiff.C

```diff
@@ -1,9 +1,9 @@
 #include "libmesh/fparser_ad.hh"
 
 // Ignore unused parameter warnings coming from cppunit headers
-#include <libmesh/ignore_warnings.h>
-#include <cppunit/extensions/HelperMacros.h>
 #include <cppunit/TestCase.h>
+#include <cppunit/extensions/HelperMacros.h>
+#include <libmesh/ignore_warnings.h>
```

Comment:
Why'd the ignore_warnings header get moved?  Placement is important for that one...

Reply:
Ugh, I did edit my `.clang-format` (which had `DisableFormat: true` in it ) to include `SortIncludes: false`. But apparently only after saving that file while working on these edits.